pkg/commands/compute/build.go

```diff
@@ -151,11 +169,31 @@ func (c *BuildCommand) Exec(in io.Reader, out io.Writer) (err error) {
 		}
 	}
 
-	if c.Manifest.File.Scripts.Build != "" {
-		lang = "custom"
+	customBuildMsg := "This project has a custom build script defined in the fastly.toml manifest. "
```

Comment:
nitpick (not a blocker): add the newline to customBuildMsg (and zap the trailing space), remove the newline from the fmt.Sprintf() below so the prompt further below becomes:

```
This project has a custom build script defined in the fastly.toml manifest.
Are you sure you want to continue with the build step? [y/N] 
```